frontend/src/employee-frontend/components/employee/EmployeePinCodePage.tsx

```diff
@@ -74,8 +80,27 @@ export default React.memo(function EmployeePinCodePage() {
       : undefined
   }
 
+  const beforeUnloadHandler = (e: BeforeUnloadEvent) => {
+    if (dirty) {
+      // Support different browsers: https://developer.mozilla.org/en-US/docs/Web/API/Window/beforeunload_event
+      e.preventDefault()
+      e.returnValue = i18n.pinCode.unsavedDataWarning
+      return i18n.pinCode.unsavedDataWarning
+    }
+    return
```

Comment:
Useless `return`

Reply:
lint complaints without it: "TS7030: Not all code paths return a value."